features/support/application_cuke_helpers.rb

```diff
@@ -17,8 +17,8 @@ def flash_message_containing?(text)
 
   def flash_message(opts={})
     selector = opts.delete(:selector)
-    selector &&= "#flash_#{selector}"
-    find(selector || '.message', {match: :first}.merge(opts))
+    selector &&= ".alert-#{selector}"
+    find(selector || '#flash-message', {match: :first}.merge(opts))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.